client/src/Components/Search/SearchPage.jsx

```diff
@@ -0,0 +1,97 @@
+import { useState, useEffect } from "react";
+import "../Search/SearchPage.css"
+import HistoryChart from "../HistoryChart";
+import SearchPageStats from "./SearchPageStats";
+import RecentTransactions from "./SearchPageRecentTransactions";
+import SearchPageNews from "./SearchPageNews";
+import Description from "./SearchPageDescription";
+import Trade from "./SearchPageTrade";
+function SearchPage(props) {
+  const search = props.search;
+  const [cryptoId, setCryptoId] = useState([]);
+  const [cryptoInfo, setCryptoInfo] = useState([]);
+
+  useEffect(() => {
+    fetch(`https://api.coingecko.com/api/v3/search?query=${search}`)
+      .then((response) => response.json())
+      .then((data) => setCryptoId(data.coins[0].id))
+      .catch((error) => console.error(error));
+  }, [search]);
+
+  console.log(cryptoId);
+  useEffect(() => {
+    fetch(`https://api.coingecko.com/api/v3/coins/${cryptoId}`)
+      .then((response) => response.json())
+      .then((data) => setCryptoInfo(data))
+      .catch((error) => console.error(error));
+  }, [cryptoId]);
+  console.log(cryptoInfo);
+  console.log(search);
+  return (
+    <div>
+      <div
+        className="avatar"
+        style={{ width: 170, height: "auto", marginLeft: 50, marginTop: 20 }}
+      >
+        <img
+          src={cryptoInfo.length === undefined && cryptoInfo.image.thumb}

```

Comment:
could you change this to `src={cryptoInfo.length === undefined ? cryptoInfo.image.thumb : undefined}` ?
error message in the console recommends to write this way